client/src/components/CreateList/CreateListForm.js

```diff
@@ -25,22 +33,25 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function CreateListForm() {
+function CreateListForm(props) {
   const [name, setName] = useState('');
 
   const dropZoneFilesRef = useRef(null);
```

Comment:
still haven't seen how useRef is used

Reply:
`useRef` is used for when you

- don't care about re-rendering that state but
- you need to persist that state across renders

A use case for `useRef` is accessing DOM nodes. Here I want to access the file located inside the DropZone component. I don't want to re-render the CreateListForm every time the file changes, but I do want to keep the value of the file for submitting the form.